### ISSUE TITLE
All processing fails with invalid argument

### ISSUE THREAD

**Issue by the-mace** [ehendrix23/tesla_dashcam#47]
OSX, Python 3.7.0:
```
$ tesla_dashcam --check_for_update
v0.1.10 is the latest release available.
MacBook-Pro:src rmason$ tesla_dashcam
GPU acceleration is enabled
Monitoring for TeslaCam Drive to be inserted. Press CTRL-C to stop
TeslaCam folder found on /Volumes/ROBS_MUSIC.
Discovered 36 folders with 307 clips to process.
	Processing 11 clips in folder /Volumes/ROBS_MUSIC/TeslaCam/SavedClips/2019-04-01_09-29-56 (1/36)
		Processing clip 1/11 from 04/01/19 09:18:57 and 60 seconds long.
			Error trying to create clip for /Volumes/ROBS_MUSIC/TeslaCam/SavedClips/2019-04-01_09-29-56/2019-04-01_09-18. RC: 1
			Command: ['ffmpeg', '-i', '/Volumes/ROBS_MUSIC/TeslaCam/SavedClips/2019-04-01_09-29-56/2019-04-01_09-18-left_repeater.mp4', '-i', '/Volumes/ROBS_MUSIC/TeslaCam/SavedClips/2019-04-01_09-29-56/2019-04-01_09-18-front.mp4', '-i', '/Volumes/ROBS_MUSIC/TeslaCam/SavedClips/2019-04-01_09-29-56/2019-04-01_09-18-right_repeater.mp4', '-filter_complex', "color=duration=60.39:s=1280x960:c=black [base];[0:v] setpts=PTS-STARTPTS, scale=640x480 , hflip [left];[1:v] setpts=PTS-STARTPTS, scale=640x480  [front];[2:v] setpts=PTS-STARTPTS, scale=640x480 , hflip [right];[base][left] overlay=eof_action=pass:repeatlast=0:x=0:y=480 [left1];[left1][front] overlay=eof_action=pass:repeatlast=0:x=320:y=0 [front1];[front1][right] overlay=eof_action=pass:repeatlast=0:x=640:y=480[tmp0];[tmp0] drawtext=fontfile=/Library/Fonts/Arial.ttf:fontcolor=white:fontsize=16.0:borderw=2:bordercolor=black@1.0:x=(w/2-text_w/2):y=(h-(text_h*2)):text='%{pts\\:localtime\\:1554124737\\:%x %X}'", '-preset', 'medium', '-crf', '28', '-allow_sw', '1', '-b:v', '5000K', '-c:v', 'h264_videotoolbox', '-y', '/Users/rmason/Movies/Tesla_Dashcam/2019-04-01_09-18.mp4']
			Error: b"ffmpeg version 4.1 Copyright (c) 2000-2018 the FFmpeg developers\n  built with Apple LLVM version 10.0.0 (clang-1000.11.45.5)\n  configuration: --prefix=/usr/local/Cellar/ffmpeg/4.1 --enable-shared --enable-pthreads --enable-version3 --enable-hardcoded-tables --enable-avresample --cc=clang --host-cflags= --host-ldflags= --enable-ffplay --enable-gpl --enable-libmp3lame --enable-libopus --enable-libsnappy --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libx264 --enable-libx265 --enable-libxvid --enable-lzma --enable-opencl --enable-videotoolbox\n  libavutil      56. 22.100 / 56. 22.100\n  libavcodec     58. 35.100 / 58. 35.100\n  libavformat    58. 20.100 / 58. 20.100\n  libavdevice    58.  5.100 / 58.  5.100\n  libavfilter     7. 40.101 /  7. 40.101\n  libavresample   4.  0.  0 /  4.  0.  0\n  libswscale      5.  3.100 /  5.  3.100\n  libswresample   3.  3.100 /  3.  3.100\n  libpostproc    55.  3.100 / 55.  3.100\nInput #0, mov,mp4,m4a,3gp,3g2,mj2, from '/Volumes/ROBS_MUSIC/TeslaCam/SavedClips/2019-04-01_09-29-56/2019-04-01_09-18-left_repeater.mp4':\n  Metadata:\n    major_brand     : mp42\n    minor_version   : 0\n    compatible_brands: mp42mp41isomiso2\n    creation_time   : 2019-04-01T13:18:57.000000Z\n  Duration: 00:01:00.38, start: 0.000000, bitrate: 4013 kb/s\n    Stream #0:0(eng): Video: h264 (Constrained Baseline) (avc1 / 0x31637661), yuv420p, 1280x960, 4010 kb/s, SAR 1:1 DAR 4:3, 36.02 fps, 10k tbr, 10k tbn, 20k tbc (default)\n    Metadata:\n      creation_time   : 2019-04-01T13:18:57.000000Z\n      handler_name    : VideoHandler\nInput #1, mov,mp4,m4a,3gp,3g2,mj2, from '/Volumes/ROBS_MUSIC/TeslaCam/SavedClips/2019-04-01_09-29-56/2019-04-01_09-18-front.mp4':\n  Metadata:\n    major_brand     : mp42\n    minor_version   : 0\n    compatible_brands: mp42mp41isomiso2\n    creation_time   : 2019-04-01T13:18:57.000000Z\n  Duration: 00:01:00.39, start: 0.000000, bitrate: 4013 kb/s\n    Stream #1:0(eng): Video: h264 (Constrained Baseline) (avc1 / 0x31637661), yuv420p, 1280x960, 4009 kb/s, SAR 1:1 DAR 4:3, 36.02 fps, 36 tbr, 10k tbn, 20k tbc (default)\n    Metadata:\n      creation_time   : 2019-04-01T13:18:57.000000Z\n      handler_name    : VideoHandler\nInput #2, mov,mp4,m4a,3gp,3g2,mj2, from '/Volumes/ROBS_MUSIC/TeslaCam/SavedClips/2019-04-01_09-29-56/2019-04-01_09-18-right_repeater.mp4':\n  Metadata:\n    major_brand     : mp42\n    minor_version   : 0\n    compatible_brands: mp42mp41isomiso2\n    creation_time   : 2019-04-01T13:18:57.000000Z\n  Duration: 00:01:00.36, start: 0.000000, bitrate: 3522 kb/s\n    Stream #2:0(eng): Video: h264 (Constrained Baseline) (avc1 / 0x31637661), yuv420p, 1280x960, 3519 kb/s, SAR 1:1 DAR 4:3, 36.02 fps, 36 tbr, 10k tbn, 20k tbc (default)\n    Metadata:\n      creation_time   : 2019-04-01T13:18:57.000000Z\n      handler_name    : VideoHandler\n[AVFilterGraph @ 0x7fceb8510e40] No such filter: 'drawtext'\nError initializing complex filters.\nInvalid argument\n"
```

Tried output to local dir but that fails:
```$ tesla_dashcam --output new "/Volumes/ROBS_MUSIC/TeslaCam"
GPU acceleration is enabled
Traceback (most recent call last):
  File "/Users/rmason/.pyenv/versions/3.7.0/bin/tesla_dashcam", line 7, in <module>
    from tesla_dashcam.tesla_dashcam import main
  File "/Users/rmason/.pyenv/versions/3.7.0/lib/python3.7/site-packages/tesla_dashcam/tesla_dashcam.py", line 2251, in <module>
    sys.exit(main())
  File "/Users/rmason/.pyenv/versions/3.7.0/lib/python3.7/site-packages/tesla_dashcam/tesla_dashcam.py", line 2109, in main
    os.mkdir(target_folder)
FileNotFoundError: [Errno 2] No such file or directory: ''```

**Comment by mrsiesta** [comment]
Looks like a configuration issue with ffmpeg:
https://stackoverflow.com/questions/48006872/no-such-filter-drawtext/48037936#48037936

**Comment by ehendrix23** [comment]
Are you using the binary one (from the DMG file) or installing from PIP? 

Asking as the binary executable includes ffmpeg whereas when installing from PIP you would need to install ffmpeg, and maybe it does not have that filter compiled in it?

I also just released 0.1.11. Although I do not think it would fix issue, worth a shot. :-)

**Comment by ehendrix23** [comment]
Closing. If issue persist please open new issue providing answer and referencing this issue.

Thank you.
